docs/acp/user/app-manifest.md

```diff
@@ -15,87 +15,92 @@ For the manifest, make sure to follow these conditions:
+| resources        | An array of objects represented as application resources (links, documents). The resource object has the following keys:<ul><li>title—name of the resource</li><li>url—full URL to the resource</li>type—type of the resource. Optional. The type affects a displayed icon. Possible values:<ul><li>internal-documentation</li><li>homepage</li><li>user-documentation</li><li>developer-documentation</li><li>release-notes</li></ul><li>fileType—type of the resource file (in case if the url leads to a file and if `type=internal-documentation`). Optional. The files are displayed directly in the AppCatalog, without redirecting to another page. Possible values:<ul><li>pdf</li><li>markdown</li></ul></ul> | See *resources example* under this table.                                                                                                                                                                                                                                                                                                                                                                                                               |
+| dependencies     | An array of objects represented as application dependencies. Dependencies array is one dimentional array where key is a dependency name, and value is it's version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | See *dependencies example* under this table.                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
it's not an array, it's an object.
```
{
  "foo/bar": "^1.0.0",
  "foo/bar2": "^2.0.0"
}
```

I don't think that it was a good idea to use this format. In the code, it will be loop on object properties, which is not the usual way of iterating the data.

I would expect something like an array of strings
```
[
   "vendor/package:^1.0.0",
   "vendor/package2:^1.0.0",
]
```
or 
```
[
   {name: "vendor/package, version: "1.0.0"},
   {name: "vendor/package2, version: "1.0.0"},
]
```

Reply:
Docs updated. The schema was discussed with @evgeniy-it. Let's not get used to that anyway as this will be supposedly refactored later.